plugins/haptics/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "tauri-plugin-haptics"
-version = "1.0.0"
+version = "2.0.0-beta.1"
```

```suggestion
version = "2.0.0-beta.0"
```
covector may not like that either but i forgot how that works for initial prereleases but we'll see 🤷